src/mongo/transport/README.md

```diff
@@ -71,17 +53,19 @@ This interface acts as a cursor to build a response message to be sent out back
 to the client.
 
 ## See Also
-For details on egress networking, see [this document][egress_networking]. For 
-details on command dispatch, see [this document][command_dispatch]. For details 
-on *NetworkingBaton* and *AsioNetworkingBaton*, see [this document][baton].
+For details on egress networking, see [Egress Networking][egress_networking]. For 
+details on command dispatch, see [Command Dispatch][command_dispatch]. For details 
+on *NetworkingBaton* and *AsioNetworkingBaton*, see [Baton][baton].
+
+For more detail about `SessionWorkflow`, see the [design][session_workflow_design].
```

Comment:
I think this document is public, but I'm not sure if our design documents are, so maybe this will be a dead-link?

Reply:
Ok using the WRITING ticket instead and marking it (internal) to reduce reader surprise.